ci-operator/step-registry/osd-ccs/e2e/aws/osd-ccs-e2e-aws-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: osd-ccs-e2e-aws
+  steps:
+    env:
+      COMPUTE_MACHINE_TYPE: "m5.xlarge"
+      MULTI_AZ: "true"
+      COMPUTE_NODES: "3"
+      ETCD_ENCRYPTION: "true"
+      MACHINE_POOL_INSTANCE_TYPE: "m5.2xlarge"
+      ENABLE_AUTOSCALING: "true"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+    pre:
+    - ref: osd-ccs-osd-ccs-admin-aws
```

Comment:
looks like this step is only generate a new AWS credential, how about rename it to, e.g. `osd-ccs/aws/create-user`, remove the prefix `osd-ccs` under `osd-ccs` folder, and make the step name expressive of its function. The remove function would be `osd-ccs/aws/delete-user`, wdyt?

Reply:
As we talked about yesterday, we may close the step.